app/views/adventures/show.html.erb

```diff
@@ -21,3 +21,7 @@
     showSource: <%= @adventure.show_source? %>
   }
 </script>
+
+<% if @adventure.theme === 'custom' %>
```

Comment:
I'm confused about the use of the subsumption operator here instead of equality. From what I know, this is basically asking "is_a? or eql?" when I think this is just trying to ask for equality. If this is intentional, please ignore.